src/internal_events/kubernetes_logs.rs

```diff
@@ -0,0 +1,31 @@
+use super::InternalEvent;
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct KubernetesLogsEventReceived<'a> {
```

Comment:
This is the only internal event the entire kubernetes source produces? What about things like error rates? Healthchecks?

Reply:
I'm open to adding new events, if you find a good spot for where an event should be emitted - let me know!

Reply:
Ok! :) Will look around.